modules/database/src/adapters/mongoose-adapter/MongooseSchema.ts

```diff
@@ -37,30 +38,22 @@ export class MongooseSchema implements SchemaAdapter<Model<any>> {
 
   async create(query: SingleDocQuery) {
     const parsedQuery = {
```

Comment:
why does create and createMany pass their docs through the parseQuery function? They do not contain operators